src/test/java/org/openmrs/contrib/qaframework/automation/RegistrationSteps.java

```diff
@@ -12,21 +11,23 @@
 import org.openmrs.reference.page.RegistrationPage;
 import org.openqa.selenium.By;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
+import io.cucumber.java.After;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
 
 public class RegistrationSteps extends Steps {
 
 	RegistrationPage registrationPage;
 	TestPatient patient;
 	ClinicianFacingPatientDashboardPage dashboardPage;
```

Comment:
You can make the class variables private since they are only consumed within the class

Reply:
Resolved thanks

Reply:
Waiting for your final reviews @kdaud.Thanks